src/huggingface_hub/hf_api.py

```diff
@@ -1964,6 +1960,18 @@ def create_commit(
                 If the Hub API returns an HTTP 400 error (bad request)
             :class:`ValueError`:
                 If `create_pr` is `True` and revision is neither `None` nor `"main"`.
+            :class: `RepositoryNotFoundError`:
```

Comment:
This is old sphinx syntax. Should probably be replaced by the new links? (same for the other values above)
```suggestion
            [`~utils.RepositoryNotFoundError`]:
```